### PR TITLE
Feature/ember update

### PR DESCRIPTION
**Pull request by njutte**
Update for Ember 2.4

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "ember-macaroni",
-  "version": "0.2.1",
-  "description": "Keep your app code DRY and copypasta free with computed property macaronis (macros)",
+  "version": "0.0.0",
```
Comment (rjhilgefort) [inline package.json]: revert this generator overwrite

File: package.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
...
+  "description": "The default blueprint for ember-cli addons.",
```
Comment (rjhilgefort) [inline package.json]: this too

File: package.json [hunk 3]
```diff
@@ -11,43 +11,38 @@
     "start": "ember server",
     "test": "ember try:testall"
   },
-  "repository": "https://github.com/poteto/ember-macaroni",
-  "bugs": "https://github.com/poteto/ember-macaroni/issues",
-  "homepage": "https://github.com/poteto/ember-macaroni",
+  "repository": "",
```
Comment (rjhilgefort) [inline package.json]: and this

File: package.json [hunk 4]
```diff
@@ -11,43 +11,38 @@
...
+    "ember-try": "^0.2.2",
+    "loader.js": "^4.0.1"
   },
   "keywords": [
-    "ember-addon",
-    "computed property macros",
-    "macros",
-    "macaroni",
-    "ember macaroni",
-    "functional macros"
```
Comment (rjhilgefort) [inline package.json:46]: revert

File: package.json [hunk 5]
```diff
@@ -11,43 +11,38 @@
...
+    "ember-addon"
   ],
   "dependencies": {
-    "ember-cli-htmlbars": "0.7.9",
```
Comment (rjhilgefort) [inline package.json:49]: don't need anymore?

Comment (njutte) [inline package.json:49]: I don't think it ever did need it as a dep I moved it into devDependencies

**Comment by rjhilgefort** [comment]
@njutte Thanks for updating.

**Pull request merged**